DeviceTests/DeviceTests.Shared/Email_Tests.cs

```diff
@@ -10,28 +10,26 @@ public class Email_Tests
     {
         [Fact]
         [Trait(Traits.InteractionType, Traits.InteractionTypes.Human)]
-        public async Task Compose_Shows_New_Window()
+        public Task Compose_Shows_New_Window()
         {
-            if (DeviceInfo.Platform == DeviceInfo.Platforms.UWP)
-                return;
-            await Email.ComposeAsync();
+            return Utils.OnMainThread(() => Email.ComposeAsync());
```

Comment:
iOS requires the UI thread, but it is probably best to do this for all platforms since it is a UI component...